app/views/govuk_publishing_components/components/docs/select.yml

```diff
@@ -0,0 +1,34 @@
+name: Select (experimental)
+description: A dropdown select
+part_of_admin_layout: true
+govuk_frontend_components:
+  - select
+accessibility_criteria: |
+  - accept focus
```

Comment:
To make this a little more readable, I'd add in something like "The component must:"

Reply:
Fixed, thanks.